plugins/scaffolder-react/src/next/components/Stepper/createAsyncValidators.ts

```diff
@@ -48,6 +63,11 @@ export const createAsyncValidators = (
           }
           formValidation[key] = fieldValidation;
         }
+      } else if (isObject(value)) {
+        formValidation[key] = (await validate(formData, path, value)) as Record<
```

Comment:
Why do we need the cast here? If needed can we use the extracted type instead?

Reply:
I'm glad you mentioned this, I've fixed the type and removed the type assertion.